vmbindings/dummyvm/benches/bench_sft.rs

```diff
@@ -0,0 +1,20 @@
+use criterion::{black_box, criterion_group, Criterion};
+
+use mmtk::plan::AllocationSemantics;
+use mmtk::vm::ObjectModel;
+use mmtk_dummyvm::api;
+use mmtk_dummyvm::test_fixtures::FixtureContent;
+use mmtk_dummyvm::test_fixtures::MutatorFixture;
+
+fn sft(c: &mut Criterion) {
+    println!("Init MMTK in sft bench");
+    let fixture = MutatorFixture::create();
+    let addr = api::mmtk_alloc(fixture.mutator, 8, 8, 0, AllocationSemantics::Default);
+    let obj = mmtk_dummyvm::object_model::VMObjectModel::address_to_ref(addr);
+
+    c.bench_function("sft read", |b| {
+        b.iter(|| api::mmtk_is_in_mmtk_spaces(black_box(obj)))
```

Comment:
It is possible. We could control what we want to benchmark. `criterion` provides an approach that allows us to microbenchmark short running functions. It is useful when we can isolate one function and check the performance of the function so that we can make changes to it with more confidence. It is a complement to what we usually do for benchmarking.

Though we can control what we want to benchmark, I am not convinced if we would like to use micro benchmarks to mimic a real situation. Using a larger benchmark or real workload is usually a better option. 

Also for this case, I don't think we want to take cache locality into consideration. For SFT access, I would like to measure the performance impact of the code change for https://github.com/mmtk/mmtk-core/pull/931. Cache locality is not what I am trying to measure, and randomly accessing memory would just add more noise to the result.